x-pack/plugins/apm/server/routes/metrics/serverless/get_compute_usage_chart.ts

```diff
@@ -25,7 +25,13 @@ import { environmentQuery } from '../../../../common/utils/environment_query';
 import { getMetricsDateHistogramParams } from '../../../lib/helpers/metrics';
 import { GenericMetricsChart } from '../fetch_and_transform_metrics';
 import { APMEventClient } from '../../../lib/helpers/create_es_client/create_apm_event_client';
-import { calcComputeUsageGBSeconds } from './helper';
+import { convertComputeUsageToGbSec } from './helper';
+
+export const computeUsageAvgScript = {
+  avg: {
+    script: `return doc['${METRIC_SYSTEM_TOTAL_MEMORY}'].value * doc['${FAAS_BILLED_DURATION}'].value`,
```

Comment:
if `system.memory.total` is not available in the document (which can happen in some cases) ES will throw a runtime error.

Reply:
This field is always available for AWS lambda metrics.